### PR TITLE
playlab playSound uses skin.soundChoices

### PR DESCRIPTION
**Pull request by cpirich**
- playlab's playSound block now uses a new `skin.soundChoices` (or `skin.soundChoicesK1`), similar to how other blocks use skin-dependent lists.
- playlab's playSound block now gets "random" as an option (must have been an oversight from long ago)
- playlab's playSound block now support `paramLists` filtering based on `level.paramRestrictions` - the dropdowns on restricted levels now match between droplet and blockly
- added loc strings for all hoc2015 sounds
- fixed sound name references in the levels to match the capitalization in `skin.soundChoices`
- fixed a bug impacting level 13 of hoc2015. the sound referenced in `startBlocks` was actually filtered out by `level.paramRestrictions` - I added the sound in question so it was no longer restricted for that level.

### CODE REVIEW

**Comment by cpirich** [comment]
@breville @islemaster

File: apps/i18n/studio/en_us.json [hunk 1]
```diff
@@ -260,6 +260,42 @@
   "playSoundWinPoint": "play win point sound",
   "playSoundWinPoint2": "play win point 2 sound",
   "playSoundWood": "play wood sound",
+  "playSoundR2D2Sound1": "play R2-D2 1 sound",
```
Comment (breville) [inline apps/i18n/studio/en_us.json:263]: would it read better to be "sound 1"?  Get that it would not be so consistent with broader usage, though.

**Comment by breville** [comment]
lgtm

File: apps/i18n/studio/en_us.json (hunk 1 above)
Comment (cpirich) [inline apps/i18n/studio/en_us.json:263]: I had it that way first, but it changes the appearance of the block - in that the word `sound` becomes part of the dropdown (where it previously was at the end after the dropdown). So, while this is a little different than droplet (which is redundant with `playSound("R2-D2sound1");)`, I think it is better to be consistent with the blockly block: `play <R2-D2 1> sound`, and that matches well with `play <random> sound` and `play <applause> sound`.

**Pull request merged**